### PR TITLE
Updates v1.1.5

### PR DESCRIPTION
**Pull request by fabriguespe**
## TLDR
- Tagline: An open framework for building AI agents ?
- Lighter but more structured and robust. 
- Updated directory
- Each agent can have its own landing in messagekit (devs can do PRs)
- Quickstarts section
- Skills: There is a 4step approach. We are calling them AI Skills ?
[Commands](https://messagekit.ephemerahq.com/skills/commands): Parsing commands with /
[Natural language](https://messagekit.ephemerahq.com/skills/natural): From @base please let swap 1 eth to usdc to /swap 1 eth usdc
[Multi Skill](https://messagekit.ephemerahq.com/skills/multi-skills): For a agent to perform multiple actions after the other
[Reasoning](https://messagekit.ephemerahq.com/skills/reasoning):  For the agent to understand all context and perform commands autonomously.
- Resolver: Fetches context from the user like ENS domain or Converse username 

### MessageKit updates
**Updates:**
   - Added `adminOnly` commands for group settings for commands enabled only for admin of groups
   - Added `memberChange` as optional runner config for enabling new members added
   - Added `attachment` as optional runner config for enabling attachments in the stream
   - New CLI logo logo
   - Updated CLI notes and warnings
   - Implemented logging for sent messages.
   - Updated `node-sdk`  https://github.com/xmtp/xmtp-js/releases/tag/%40xmtp%2Fnode-sdk%400.0.20
   - Updated `xmtp-js` https://github.com/xmtp/xmtp-js/releases/tag/%40xmtp%2Fxmtp-js%4013.0.3
   - Updated grpc-client https://github.com/xmtp/xmtp-node-js-tools/releases
   - Created Converse usernames endpoint. https://github.com/ephemeraHQ/converse-website/pull/32
   - Revamped command handler and triggers.
   - Added logging for invalid commands.
   - Added handling for incorrect and non-command inputs.
   - New Tagline: `An open framework for building AI agents`
   - Updated tests
   - Improved `isMessageValid` for filtering messages
  - Added `isMessageValid` log to debug on `MSG_LOG=true`
  - Added `isTagged` in the filter
  - Restructured the isMessageValid logic
  - Updated streams with reconnect logic
  - Updated types based on Skills

**Documentation:**
   - Restructured directory
   - Added quickstarts.
   - Replaced one-to-one with Agent example
   - Documented `skills`
   - Renamed examples as templates
   - How to use a second `xmtpClient` client
   - Optional parameters concepts/structure#optional-parameters
   - Added changelog

   
**Middleware:**
Added Middlewares section as the layer where to interact with external services in general.
   - GPT: Use gpt responses (Updated)
   - Cron: Send updates daily
   - Resolver: Resolve ens and converse usernames
   - Notion: Interact with notion docs
   - Redis: Interact with Redis DB
   - Stackso: Use stackso for tracking points and loyalty programs
   - Vision: Use GPT vision to parse attachments
   - Learnweb3: Use learnweb3 libraries for testnet funds
   
**Bug Fixes:**
   - Fixed CLI issues when install with npm `npm init message-kit@latest`

### CODE REVIEW

File: packages/docs/pages/index.mdx [hunk 1]
```diff
@@ -31,45 +31,65 @@ yarn create message-kit
...
+
 :::
 
 </div>
   <CustomHomePage.TileGrid>
+
+    <CustomHomePage.Tile
+      href="/directory"
+      title="Directory"
+      description="Mmost popular Apps built with MessageKit"
```
Comment (saulmc) [inline packages/docs/pages/index.mdx:46]:
```suggestion
      description="Most popular Apps built with MessageKit"
```

File: packages/docs/pages/middlewares/notion.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
+## Cron middleware
```
Comment (saulmc) [inline packages/docs/pages/middlewares/notion.mdx:1]:
```suggestion
## Notion middleware
```

File: packages/create-message-kit/index.js [hunk 3]
```diff
@@ -14,12 +14,21 @@ const packageJson = JSON.parse(
...
-    intro(pc.red(`Welcome to MessageKit CLI v${version}!`));
+    log.info(pc.cyan(`Welcome to MessageKit CLI v${version}!`));
+    const coolLogo = `
+███╗   ███╗███████╗███████╗███████╗ █████╗  ██████╗ ███████╗██╗  ██╗██╗████████╗
+████╗ ████║██╔════╝██╔════╝██╔════╝██╔══██╗██╔════╝ ██╔════╝██║ ██╔╝██║╚══██╔══╝
+██╔████╔██║█████╗  ███████╗███████╗███████║██║  ███╗█████╗  █████╔╝ ██║   ██║   
+██║╚██╔╝██║██╔══╝  ╚════██║╚════██║██╔══██║██║   ██║██╔══╝  ██╔═██╗ ██║   ██║   
+██║ ╚═╝ ██║███████╗███████║███████║██║  ██║╚██████╔╝███████╗██║  ██╗██║   ██║   
+╚═╝     ╚═╝╚══════╝╚══════╝╚══════╝╚═╝  ╚═╝ ╚═════╝ ╚══════╝╚═╝  ╚═╝╚═╝   ╚═╝   
+Powered by XMTP`;
```
Comment (rygine) [inline packages/create-message-kit/index.js:29]: nit: this is indeed cool, but it's a bit hard to read. is there another text style that would work better?

File: packages/message-kit/src/lib/runner.ts [hunk 4]
```diff
@@ -43,105 +39,225 @@ export default async function run(handler: Handler, config?: Config) {
...
+
+    while (true) {
+      try {
+        if (STREAM_LOG) {
+          console.log(
+            `[${version}] Attempting to start client stream... (Attempt ${retryCount + 1})`,
+          );
+        }
+
         try {
```
Comment (rygine) [inline packages/message-kit/src/lib/runner.ts:221]: why nested `try..catch` blocks?

**Review by rygine — APPROVED** [review]
